app/views/maps/_token_drawer.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="current-map__token-drawer token-drawer flex flex-row items-center justify-start select-none bg-gray-500 mx-4 p-2 rounded-lg border whitespace-no-wrap overflow-x-auto overflow-y-hidden shadow-lg" data-target="map--tokens.drawer toggle.content" data-action="dragover->map--tokens#dragOverDrawer drop->map--tokens#dropOnDrawer open->toggle#open close->toggle#close" data-toggle-target="token-drawer">
```

Comment:
Tailwind + Stimulus makes for some very long tags. The advantage of inspecting the DOM to look at the current state is impeded a bit as a result. Not for this PR obviously but I wonder if I should put some time into refactoring the styling